ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Overview/WorkloadCvesOverviewPage.tsx

```diff
@@ -457,129 +371,95 @@ function WorkloadCvesOverviewPage() {
-                                    onFilterChange={setSearchFilter}
-                                    filterToolbar={filterToolbar}
-                                    entityToggleGroup={entityToggleGroup}
-                                    rowCount={entityCounts.Image}
-                                    sort={sort}
-                                    workloadCvesScopedQueryString={workloadCvesScopedQueryString}
-                                    isFiltered={isFiltered}
-                                    pagination={pagination}
-                                    hasWriteAccessForWatchedImage={hasWriteAccessForWatchedImage}
-                                    onWatchImage={(imageName) => {
-                                        setDefaultWatchedImageName(imageName);
-                                        watchedImagesModalToggle.openSelect();
-                                        analyticsTrack(WATCH_IMAGE_MODAL_OPENED);
-                                    }}
-                                    onUnwatchImage={(imageName) => {
-                                        setUnwatchImageName(imageName);
-                                        unwatchImageModalToggle.openSelect();
-                                    }}
-                                    imageTableColumnOverrides={{
-                                        cvesBySeverity: hideColumnIf(!isViewingWithCves),
-                                        rowActions: hideColumnIf(
-                                            !hasWriteAccessForWatchedImage &&
-                                                !hasWriteAccessForImage
-                                        ),
-                                    }}
-                                />
-                            )}
-                            {activeEntityTabKey === 'Deployment' && (
-                                <DeploymentsTableContainer
-                                    searchFilter={searchFilter}
-                                    onFilterChange={setSearchFilter}
-                                    filterToolbar={filterToolbar}
-                                    entityToggleGroup={entityToggleGroup}
-                                    rowCount={entityCounts.Deployment}
-                                    pagination={pagination}
-                                    sort={sort}
-                                    workloadCvesScopedQueryString={workloadCvesScopedQueryString}
-                                    isFiltered={isFiltered}
-                                    deploymentTableColumnOverrides={{
-                                        cvesBySeverity: hideColumnIf(!isViewingWithCves),
-                                    }}
-                                />
-                            )}
-                        </CardBody>
-                    </Card>
-                </PageSection>
+                <VulnerabilitiesOverview
```

Comment:
Thinking about reuse, what are pro and con:
* Lift up `PageSection` element from component to this level?
    Especially since I noticed at the end of the changed lines: `PageSection` inside `PageSection` which we might eventually want to untangle.
* Let component render `Card` as the reusable layout element?

Reply:
Yes this is a great idea. The only main con is that we might need to duplicate the wrapper when used from the plugin side, but I think we will likely get rid of the wrapper soon anyway.

Since it looks like the plugin and PF6 are landing at the same time, it would be nice if we could untangle some of these nested layouts and get consistency with the console as a bonus when the styling changes due to the PF upgrade.